.eslintrc.json

```diff
@@ -1,13 +1,25 @@
 {
   "extends": "airbnb-base",
   "env": {
-    "node": true
+    "node": true,
+    "mocha": true
```

Comment:
Put the specific rules that apply to all tests under the tests directory as a separate `.eslintrc.json`.

Reply:
I have never seen that in any Mojaloop repository so far. Is there a strong reason for doing so?

Reply:
Probably because I haven't touched enough of the repositories. The strong reason for doing so is that you don't want to use mocha globals in anything other than test files, and you don't want to use the dangling modifiers with anything other than `__rewire__`. Why would you change the ruleset for the whole project just because of those things? You just expand the surface area of where problems can creep in. I'd actually prefer you only disable the dangling rules _within_ test files that specifically use rewire and need it.